cmd/clairctl/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+- 1.5
```

Comment:
Remove this whole file and combine the building and testing with the root .travis.yml. I'm not familiar with the deploy stuff. Is that necessary?